.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -22,7 +28,7 @@ Add a changelog entry for this PR by answering the following questions:
       - [ ] Terraform
       - [ ] No refresh required for updates in this PR
 
-### Contribution checklist:
+### Contribution checklist (MS Employees Only):
 - [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
```

Comment:
The "bcforapi" short link points to an outdated version of the breaking changes doc. I suggest using the link that Jeffrey/Srikanth maintain to always point to the latest version.
```suggestion
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/AzBreakingChangesPolicy) of "no breaking changes"
```